modules/client/CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ TEMPLATE:
 
 ## [UPCOMING]
 
+### Fixed
+- Added missing security check that checks that the to address in the permission actions is the DAO address
```

Comment:
```suggestion
- Added missing security check that checks that the `to` address in the permission actions is the DAO address
```
Use appropriate formatting. Otherwise, it isn't clear what which words are variable/function name and which are regular text.